scripts/nb-tester/qiskit_docs_notebook_tester/__init__.py

```diff
@@ -22,30 +26,27 @@
 import nbclient
 import nbconvert
 import nbformat
+import tomli as tomllib
 from qiskit_ibm_runtime import QiskitRuntimeService
 from squeaky import clean_notebook
 
-NOTEBOOKS_GLOB = "[!.]*/**/*.ipynb"
-NOTEBOOKS_EXCLUDE = [
-    "docs/api/**",
-    "**/.ipynb_checkpoints/**",
-]
-NOTEBOOKS_THAT_SUBMIT_JOBS = [
-    "docs/start/hello-world.ipynb",
-    "tutorials/build-repitition-codes/build-repitition-codes.ipynb",
-    "tutorials/chsh-inequality/chsh-inequality.ipynb",
-    "tutorials/grovers-algorithm/grovers.ipynb",
-    "tutorials/quantum-approximate-optimization-algorithm/qaoa.ipynb",
-    "tutorials/repeat-until-success/repeat-until-success.ipynb",
-    "tutorials/submitting-transpiled-circuits/submitting-transpiled-circuits.ipynb",
-    "tutorials/variational-quantum-eigensolver/vqe.ipynb",
-]
+@dataclass
+class Config:
+    all_notebooks: str
+    notebooks_exclude: list[str]
+    notebooks_that_submit_jobs: list[str]
 
+    @classmethod
+    def read(cls, path: str) -> Config:
+        """
+        Load the globs from the TOML file
+        """
+        return cls(**tomllib.loads(Path(path).read_text()))
```

Comment:
Warning this will be a cryptic error if the three options are not set:

```
>>> from dataclasses import dataclass
>>> @dataclass
... class Foo:
...   a: str
...
>>> Foo(**{})
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __init__() missing 1 required positional argument: 'a'
```

Imo that's fine though because this is only going to be used in our two repos that we tightly control. I don't expect content writers to be changing the keys in the TOML file. So we can keep it simple.

Reply:
I've added a more helpful error message for that case